app/kumactl/cmd/install/testdata/install-demo.defaults.golden.yaml

```diff
@@ -91,3 +91,49 @@ spec:
           ports:
             - name: http
               containerPort: 5000
+---
+apiVersion: kuma.io/v1alpha1
+kind: MeshGateway
+mesh: default
+metadata:
+  name: demo-app
+spec:
+  conf:
+    listeners:
+    - port: 80
+      protocol: HTTP
+  selectors:
+  - match:
+      kuma.io/service: demo-app_gateway
+---
+apiVersion: kuma.io/v1alpha1
+kind: MeshGatewayInstance
+metadata:
+  name: demo-app-gateway
+  namespace: kuma-demo
+spec:
+  replicas: 1
+  serviceType: LoadBalancer
```

Comment:
This works without problems in k3d even if there's no LB types?

Reply:
If there's no LB support then it just behaves like a ClusterIP service.

Reply:
https://github.com/kumahq/kuma-counter-demo/pull/29